crates/sui-indexer/src/handlers/checkpoint_handler.rs

```diff
@@ -228,6 +220,12 @@ where
                 let mutated_object_chunk = chunk.1.into_iter().zip(object_datas);
                 acc.extend(mutated_object_chunk);
                 Ok::<_, Error>(acc)
+            })
+            .map_err(|e| {
+                IndexerError::SerdeError(format!(
+                    "Failed to generate changed objects with err {:?}",
```

Comment:
log `tx.digetst`

Reply:
this is about all objects of a checkpoint, logging checkpoint seq instead.